### DISCUSSION TITLE
Can we auto generate some tests ?

### DISCUSSION THREAD

**Discussion by adrien-zinger** [massalabs/massa#1949] [Ideas]
I open this discussion to speak about tests in the execution module. Some tests are really similar and could be generated by something else than a developer.

> _Is there a way to auto generate tests that are just asserting that a command is propagated, not only here but every where else?_ - @AureliaDolo 

Example of common test in several modules.

https://github.com/massalabs/massa/blob/cc24729d9f2223d9d900ae70e3b614089e1d011a/execution/src/tests/scenarios_mandatories.rs#L21-L32

**Comment by AureliaDolo** [comment]
Related to #1919
